docs/sftt/sftt-api-spec.md

```diff
@@ -294,11 +351,11 @@ Invite someone to join a team. Will send an email to all specified people that i
 
 ##### `200 OK`
 
-!!! missing "Unknown"
+Users invited.
 
 ##### `400 BAD REQUEST`
 
-!!! missing "Should be an error here if the team id is invalid or if the user is already on the team"
+If the team id is invalid or if the one of the users is already on the team. 
```

Comment:
This seems problematic because there's no way for the caller to know _which_ user caused the error... I'd say we should do 1 of 3 things:
1. Just ignore users already on the team and return a 200 response regardless
2. If any of the users are already on the team then don't send any of the invites and return a 400
3. Add a response body to the message that shows the id of the users that are already on the team. In this case we should make the status code 409 CONFLICT so that we don't have to also deal with the team id being invalid

Personally I'd go for 1. It's pretty easy to implement and if a user wants someone on their team and invites them then they've basically already accomplished their goal if the user is on the team already so a 200 seems reasonable. Up to you which way to go though

Reply:
Implemented 1